exonum/src/storage/db.rs

```diff
@@ -22,8 +23,78 @@ use super::Result;
 
 use self::NextIterValue::*;
 
+/// Map containing changes with corresponding key.
+/// Changes
```

Comment:
Is this line needed (you will see in the API docs in an IDE a sentence followed by a single word)?